1 table sales.sql

```diff
@@ -0,0 +1,42 @@
+use BikeShopDB
+go
+drop table if exists Sales
+go
+create table dbo.Sales(
+	SaleId int not null identity primary key,
+	CustomerName varchar (30) not null
+		constraint c_sales_customer_name_cannot_be_blank check(CustomerName > ''),   
+	CustomerAddress varchar (70) not null, 
+		constraint c_sales_customer_address_cannot_be_blank check(CustomerAddress > ''),   
+	CustomerPhoneNumber varchar (16) not null
+		constraint c_sales_customer_phone_number_cannot_be_blank check(CustomerPhoneNumber > ''),
+		constraint c_sales_customer_phone_number_must_be_numeric check(CustomerPhoneNumber not like '%[^0-9]%'),   
+	BikeCompany varchar (15) not null
+		constraint c_sales_company_cannot_be_blank check(BikeCompany > ''), 
+	BikeSize char (3) not null

```

Comment:
Rather than storing it as a char to include the inches symbol, store it as an int and include that in the column name. ie: SizeInInches